pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -23,7 +23,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.25.4"
+	CustomResourceDefinitionSchemaVersion = "1.25.3"
```

Comment:
:thinking:

Reply:
I noticed the patch version was bumped two times:
```
$ git diff v1.12.0-rc2 HEAD pkg/k8s/apis/cilium.io/ | grep CustomResourceDefinitionSchemaVersion
-       CustomResourceDefinitionSchemaVersion = "1.25.2"
+       CustomResourceDefinitionSchemaVersion = "1.25.4"
        // CustomResourceDefinitionSchemaVersionKey is key to label which holds the CRD schema version
        CustomResourceDefinitionSchemaVersionKey = "io.cilium.k8s.crd.schema.version"
-       CustomResourceDefinitionSchemaVersion = "1.25.2"
+       CustomResourceDefinitionSchemaVersion = "1.25.4"
        // CustomResourceDefinitionSchemaVersionKey is key to label which holds the CRD schema version
        CustomResourceDefinitionSchemaVersionKey = "io.cilium.k8s.crd.schema.version"
```

Reply:
Would have been nice to move this to its own commit with explanation for context.